locks.go

```diff
@@ -124,21 +124,21 @@ type LockRedis struct {
 }
 
 // NewLockRedis creates a new instance of LockRedis.
-func NewLockRedis(pool redsyncredis.Pool, mutexName string) *LockRedis {
+func NewLockRedis(pool redsyncredis.Pool, mutexName string, oo ...redsync.Option) *LockRedis {
```

Comment:
Nit: could you please rename `oo` to `options` or smth like that so that it's self explanatory? 
Thanks!

Reply:
Sure!
Done.